apps/smartling/lambda/src/index.ts

```diff
@@ -107,11 +107,7 @@ export function makeApp(fetchFn: any, issuer: any) {
     );
   });
 
-  if (process.env.LOCAL_DEV === 'true') {
-    app.use('/frontend', express.static(path.resolve(__dirname, '../../frontend/build')));
-  } else {
-    app.use('/frontend', express.static(path.resolve(__dirname, '../../static')));
-  }
+  app.use('/frontend', express.static(path.resolve(__dirname, '../node_modules/smartling-frontend/build')));
```

Comment:
why removed? could be still used in development

Reply:
no with `lerna bootstrap` you have it symlinked in node_modules in dev as well